Samples/ExtendedControlAndMetadata/EyeGazeAndBackgroundSegmentation/README.md

```diff
@@ -7,7 +7,7 @@ This project also contains a helper method to extract and deserialize frame meta
 
 ## Requirements
 	
-This sample is built using Visual Studio 2019 and requires [Windows SDK version 22000](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK).
+This sample is built using Visual Studio 2019 and requires [Windows SDK version 22000](https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/) at the very least.
```

Comment:
>requires [Windows SDK version 22000] [](http://example.com/codeflow?start=50&length=36)

I think this will actually now fail with 22000 as I removed most of the redefined values from the KSHelper.h as the the 22621 SDK has access to them now. 

I think we should start using Git tags or so to mark the spot for master branch where it builds with certain SDK release.

Thoughts ?

Reply:
hmm that sample specifically builds and run fine with that SDK as target, I think this is ok; There are other DDIs that we have defines for that yeah would require newer SDK but they are not covered in this sample

Reply:
pushed a change to make 22000 the min target, for extendedprop sample all the redefinition I need are in the .idl so we are good to go this way I think